db/migrate/20181218015155_converttosti.rb

```diff
@@ -0,0 +1,5 @@
+class Converttosti < ActiveRecord::Migration[5.2]
```

Comment:
its good practice to put the migrations in their own commit. Since there is possibility it could ruin things.